cd_scripts/install_starter_script.sh

```diff
@@ -0,0 +1,87 @@
+#! /bin/bash
+set -x
+gsutil cp  gs://gcsfuse-release-packages/version-detail/details.txt .
+curl http://metadata.google.internal/computeMetadata/v1/instance/name -H "Metadata-Flavor: Google" >> details.txt
+touch ~/logs.txt
+
+# Install apt-transport-artifact-registry
+if grep -q ubuntu details.txt || grep -q debian details.txt;
+then
+    curl https://us-central1-apt.pkg.dev/doc/repo-signing-key.gpg | sudo apt-key add - && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+    echo 'deb http://packages.cloud.google.com/apt apt-transport-artifact-registry-stable main' | sudo tee -a /etc/apt/sources.list.d/artifact-registry.list
+    sudo apt update
+    sudo apt install apt-transport-artifact-registry
+    echo 'deb ar+https://us-apt.pkg.dev/projects/gcs-fuse-prod gcsfuse-bullseye main' | sudo tee -a /etc/apt/sources.list.d/artifact-registry.list
+    sudo apt update
+
+    # Install released gcsfuse version
+    yes | sudo apt install gcsfuse=$(sed -n 1p details.txt) -t gcsfuse-bullseye |& tee -a ~/logs.txt
```

Comment:
Why do we need to specify target release and a fixed one (gcsfuse-bullseye) ?

Reply:
Changed to $(lsb_release -cs) for target release